### PR TITLE
Refactored Bridge events fields

### PR DESCRIPTION
**Pull request by kelvinator07**
## Description

- Refactored Solidity Type Constants

- Refactored Fields Constants

### CODE REVIEW

**Review by josedahlquist — DISMISSED** [review]

**Comment by josedahlquist** [comment]
pipeline:run

File: rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
     public static final String INT = "int";
     public static final String INT256 = "int256";
     public static final String UINT = "uint";
+    public static final String UINT256 = "uint256";
```
Comment (pamgonzalez) [inline rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java:45]: Are you sure that we don't need any modification in static class IntType?

**Review by pamgonzalez — APPROVED** [review]

**Review by pamgonzalez — DISMISSED** [review]

File: rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java (hunk 1 above)
Comment (kelvinator07) [reply rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java:45]: I Have Checked Thoroughly... No Other Modification Needed

If The getCanonicalName() Gets Called.. The Parent SolidityType Returns The name, Which I Think Is Fine

**Review by josedahlquist — DISMISSED** [review]

**Review by marcos-iov — APPROVED** [review]

**Comment by josedahlquist** [comment]
pipeline:run

**Review by ajlopezrsk — APPROVED** [review]

**Comment by josedahlquist** [comment]
pipeline:run

**Comment by josedahlquist** [comment]
pipeline:run

**Review by josedahlquist — APPROVED** [review]

**Comment by marcos-iov** [comment]
pipeline:run

**Review by ajlopezrsk — APPROVED** [review]

**Pull request merged**